requirements/requirements.in

```diff
@@ -24,6 +24,7 @@ marshmallow_enum
 matrix-client==0.3.2
 mirakuru==2.1.2
 netifaces
+packaging
```

Comment:
Do we need this in the runtime requirements?
Shouldn't `-dev` or `-ci` be enough?

Reply:
Unfortunately, the nightly build fails when building the binaries because we don't have this in the general requirements.

Reply:
Right. We intentionally don't install the dev dependencies while building the bundle.

I'd say we could add it here:
https://github.com/raiden-network/raiden/blob/542e96aa79db4016f4201799eb521de02dfc6ec1/docker/build.Dockerfile#L26

Or add a new requirements file (e.g. `requirements-build.in`) that gets used while building the bundles.

Reply:
> Right. We intentionally don't install the dev dependencies while building the bundle.

Hm, but is this really a dev dependency? I mean, we use it in `raiden.utils.ethereum_clients` and now, with my changes to fix the typing errors, we use it even more. There's both `pkg_resources` and `packaging` which does not look ideal, so maybe we should just decide on one?